docs/core/extensions/channels.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Channels in .NET
+description: Learn how to use System.Threading.Channels in .NET.
+author: IEvangelist
+ms.author: dapine
+ms.date: 08/02/2022
+---
+
+# System.Threading.Channels in .NET
+
+<xref:System.Threading.Channels?displayProperty=fullName> provides a set of synchronization data structures for passing data between producers and consumers asynchronously. It's a library targeting .NET Standard that works on all .NET implementations.
+
+This library is available in the [System.Threading.Channels](https://www.nuget.org/packages/System.Threading.Channels) NuGet package (or included when targeting .NET Core 2.1+).
+
+## Producer/Consumer conceptual programming model
+
+Channels are an implementation of the producer/consumer conceptual programming model. In the producer/consumer programming model, producers asynchronously produce data, and consumers asynchronously consume that data. In other words, this model hands off data from one party to another. Try to think of channels as you would any other common generic collection type, such as a `List<T>`. The primary difference is that this collection manages synchronization and provides various consumption models through factory creation options. These options control the _bounding strategies_ employed by the channel instance and the behavior of the channel when the bound is reached.
+
+## Bounding strategies
+
+Depending on how a `Channel<T>` is created, the <xref:System.Threading.Channels.Channel%602.Writer?displayProperty=nameWithType> and <xref:System.Threading.Channels.Channel%602.Reader?displayProperty=nameWithType> will behave differently.
```

Comment:
```suggestion
Depending on how a `Channel<T>` is created, its reader and writer will behave differently.
```

Reply:
I changed this because the previous wording makes it sounds like the *properties* behave differently, which I don't think they do?